keyboards/handwired/polykybd/lang/_gen_batch3_cols.py

```
@@ -0,0 +1,151 @@
+the FIVE tone marks (combining grave/hook/tilde/acute/dot-below) on keys 5-9 -
+those combining marks are emitted as dotted-circle composites (fontconvert -C)
+just like the Indic/Thai matras.
+
+Outputs: /tmp/b3_cols.json, /tmp/b3_named.json, /tmp/b3_fonts.json
+"""
+import json
+from fontTools.ttLib import TTFont
+F="/home/user/qmk_firmware/keyboards/handwired/polykybd/fonts/"
+NS=set(TTFont(F+"noto-sans/NotoSans-Regular.ttf").getBestCmap().keys())
+
+ROW={}; L="ABCDEFGHIJKLMNOPQRSTUVWXYZ"
```

Ruff (0.15.15)

[error] Multiple statements on one line (semicolon)

(E702)